packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -211,20 +223,31 @@
           <button class="control-button diagram-reload" @click="resetDiagram" title="Clear">
             <ReloadIcon class="control-button__icon" />
           </button>
-          <v-popper
-            v-if="appMapUploadable"
-            class="hover-text-popper"
-            text="Create a link to this AppMap."
-            placement="left"
-            text-align="right"
-          >
-            <button class="control-button appmap-upload" @click="uploadAppmap" title="">
-              <UploadIcon class="control-button__icon" />
-              <span>Share</span>
-            </button>
-          </v-popper>
         </template>
       </v-tabs>
+
+      <div v-if="showShareModal" class="share-appmap">
+        <div class="heading">
+          <h1>Share this AppMap</h1>
+          <span class="close-me" @click="closeShareModal">X</span>
+        </div>
+        <div class="content">
+          <p>
+            This link can be used to view this AppMap in a browser. You can share it with other
+            collaborators, or put it into a Pull Request.
+          </p>
+          <div class="share-url">
+            <div class="share-link">
+              <span class="url">{{ shareURLmessage }}</span>
+              <span class="icon copy" @click="copyToClipboard(shareURL)">
+                <CopyIcon />
+              </span>
+            </div>
+            <p>This link has been copied to your clipboard.</p>
```

Comment:
I don't actually see anywhere where the link is automatically copied to the clipboard. Is this supposed to be conditional or added to a popper?

Reply:
I made it so that the text gets copied to the clipboard from VS Code [here](https://github.com/getappmap/vscode-appland/pull/574/files#diff-33c690dfe00cdf7992db7a1d2dba67112767481b40edbac481450b4fcb58d2e2R219).